static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -53,6 +54,7 @@ export const getEpicTestToRun = memoize(
                 );
 
                 return firstRunnableTest<EpicABTest>([
+                    hardCodedPriorityEpicTest,
```

Comment:
I think this name makes this part even more confusing. Perhaps just reference `remoteRenderEpic` directly here, as it's a temporary special case?

Reply:
@tomrf1 yeah I agree. I only introduced this indirection to make the tests pass to be honest, but can explain IRL as would definitely prefer a better solution too!